test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/transforms/DeferredFieldIsRenamedTestSnapshot.kt

```diff
@@ -0,0 +1,74 @@
+// @formatter:off
```

Comment:
This is the baseline test we are trying to get working

(currently, the expected result is wrong)